api/Credential.json

```diff
@@ -173,9 +179,9 @@
             }
           },
           "status": {
-            "experimental": true,
-            "standard_track": true,
-            "deprecated": false
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
```

Comment:
I realize representing mixins is a bit odd (#472), but this status seems off to me: https://w3c.github.io/webappsec-credential-management/#credentialuserdata Maybe this data needs to be moved? Or returned to experimental and standard track?

Reply:
To be honest, I wasn't really aware of the mixin situation.

Strictly speaking, `CredentialUserData` is not included in `Credential` itself, but in the only two interfaces that extend it (i.e. `FederatedCredential` and `PasswordCredential`).

Would it be a bad idea to extract `name` to a dedicated `CredentialUserData` feature (alongside with the – anyway – missing `iconURL` sub-feature)?

Reply:
> Would it be a bad idea to extract name to a dedicated CredentialUserData feature (alongside with the – anyway – missing iconURL sub-feature)?

Well, the discussion on #472 makes it sound like the right way to go is to document mixin interfaces as if they were part of the interface that inherits from the mixin. I get the impression that mixins are a sort of shorthand for spec authors to say that two or more interfaces will have some semantics in common, not that browsers will necessarily implement them as named mixins.

So in this case, I'd expect to see features for `Credential.name`, `FederatedCredential.name`, and `PasswordCredential.name`, assuming that there's actually implementations out there.

Reply:
Alright. Should we maybe describe the `name` sub-feature with `<code>CredentialUserData.name</code>` to make it easier to understand that this is not *directly* a sub-feature of `FederatedCredential` and `PasswortCredential`?

Reply:
Yeah, I'm with you that mixins are confusing. I like your `includes <code>FooMixin</code>` idea on #472 — that seems like a promising approach. But I don't think I'm the right person to make a decision about it (Florian is the right person, though he's away on vacation right now). But I'd rather not delay merging this PR. What do you think of this:

- Set the standards track data like I suggested above (true, true, false)
- Set the description to `"<code>name</code> (from <code>CredentialUserData</code> mixin)"` (building on your idea, with a more explicit description of the relationship between `CredentialUserData` and `Credential`)
- Revisit the discussion on #472 when Florian gets back (in a couple weeks)

Does that sound OK to you?

Reply:
Thank you very much, let's do this (see ccdd0dac725018ebdebe11319a219f67abad8d38)! :-)